src/emulator/functionsEmulator.ts

```diff
@@ -551,7 +551,7 @@ export class FunctionsEmulator implements EmulatorInstance {
    *  specified in extension.yaml. This will ALWAYS be populated when emulating extensions, even if they
    *  are using the default version.
    */
-  async askInstallNodeVersion(cwd: string, nodeMajorVersion?: string): Promise<string> {
+  askInstallNodeVersion(cwd: string, nodeMajorVersion?: string): Promise<string> {
```

Comment:
Seems like this function should just return `string` not `Promise<string>`

Reply:
Sure. Fixed. Was avoiding updating code but looks like it's all in this file.